libs/ngxtension/derived-async/src/derived-async.ts

```diff
@@ -271,9 +272,7 @@ export function derivedAsync<T>(
 				untracked(() => sourceEvent$.next(newSource));
 			} else {
 				// if the new source is not an observable or a promise, we set the value immediately
-				untracked(() =>
-					sourceValue.set({ kind: StateKind.Value, value: newSource as T }),
-				);
+				untracked(() => sourceEvent$.next(of(newSource as T)));
 			}
```

Comment:
Hi @MillerSvt 
We can refactor it a bit more now to just be: 
```ts
// we untrack the source$.next() so that we don't register other signals as dependencies
untracked(() => {
  sourceEvent$.next(
    isObservable(newSource) || isPromise(newSource) ? newSource : of(newSource as T)
  );
});
```